kolibri/core/assets/src/components_demo.js

```diff
@@ -109,6 +109,160 @@ var TextInputWithTagDisplay = Mn.LayoutView.extend({
     }
 });
 
+
+// This is a deeply nested view, used below.
+// Marionette requires deeply-nested views to be defined before they're used in their containing views.
+var ClassroomView = Mn.LayoutView.extend({
+    template: function(serialized_model) {
+        var html = _.join(['<span>', serialized_model.name, ':', '</span>'], '');
+        html += '<ul class="ko_list userList">';
+        _.forEach(this.users, function(user){
```

Comment:
Would we be able to clean this up using some more of [_.forEach](https://lodash.com/docs#forEach)'s functionality? Or does marionette not like that?

'_.forEach({ 'a': 1, 'b': 2 }, function(value, key) {
  console.log(key);
});'